presto-server-rpm/src/main/rpm/postremove

```diff
@@ -3,9 +3,10 @@
 # if this is the last version of presto-server-rpm being removed (i.e. not on upgrade)
 if [ "$1" -eq 0 ]
 then
-    # Delete the conf directory manually during uninstall.
-    # rpm -e wont remove it, because this directory was manually updated in postinstall
-    rm -rf /etc/presto
+    # Delete /etc/presto/env.sh manually during uninstall.
+    # rpm -e will not remove since it's not considered a configuration file
```

Comment:
```suggestion
    # rpm -e will not remove since it's considered a configuration file
```
?

Reply:
No. If it would be configuration file then RPM would take care of this.

Reply:
I had made it simler, matched the other commented message. 

```
# rpm -e wont remove it, because this file is created during preinstall
```